pkgs/development/libraries/wolfssl/default.nix

```diff
@@ -5,37 +5,55 @@
 , autoreconfHook
 , util-linux
 , openssl
+, variant ? "all"
+, extraFlags ? [ "--enable-pkcs11" "--enable-writedup" ]
```

Comment:
Those flags here should get some comment. Also why are they disabled when anything is supplied?

Reply:
Are the comments/explaination acceptable to you now?

Reply:
Moved the 'default' flags to the actual derivation like the rest, as it's only useful in the 'all' variant anyway.
Instead allows `extraConfigureFlags` like the name implies.